### PR TITLE
Updated warning for boundary.circle

### PR DESCRIPTION
**Pull request by avulfson17**
Fixes (or at least modifies) #510

### CODE REVIEW

**Comment by orangejulius** [comment]
LGTM!

File: test/unit/sanitiser/_geo_reverse.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ module.exports.tests.sanitize_boundary_country = function(test, common) {
     // t.equals(clean['boundary.circle.radius'], 12.121212, 'should be set to point.lat')
     t.deepEquals(errorsAndWarnings, {
       errors: [],
-      warnings: ['boundary.circle is currently unsupported']
+      warnings: []
```
Comment (dianashk) [inline test/unit/sanitiser/_geo_reverse.js]: this test is explicitly checking for a warning that `boundary.circle.radius` is going to be ignore, which it is. the code changes you've made don't (or shouldn't) change that behavior. this test is actually proving that the code is broken.

Comment (dianashk) [inline test/unit/sanitiser/_geo_reverse.js]: sorry, ignore the first comment. i was reading this wrong.

however, the name of the test should be updated to reflect what the new expectation is.

**Comment by dianashk** [comment]
:shipit:

**Pull request merged**